addons/website_event/tests/test_event_internals.py

```diff
@@ -224,3 +226,87 @@ def test_registration_answer_search(self):
         # should fetch "registration_2" because the answer to the first question is "Q1-Answer2"
         # should fetch "registration_3" because the answer to the third question is "Answer2" (as free text)
         self.assertEqual(search_res, registration_2 | registration_3)
+
+    @users('portal')
+    def test_portal_user_is_visible_on_website(self):
+        """
+        Tests the computed field `is_visible_on_website` on `event.event`.
+        `filtered_domain` is used to check if the `_compute_is_visible_on_website` method is correct
```

Comment:
Docstring seems outdated, would say it's overkill as it's pretty clear from method name, we could remove to keep it short.